docs/frameworks/symfony.md

```diff
@@ -154,46 +103,51 @@ To learn more about all this, read the [environment variables documentation](/do
+  # trust "X-Forwarded-*" headers coming from API Gateway
+  trusted_headers: [ 'x-forwarded-for', 'x-forwarded-proto', 'x-forwarded-port' ]
 ```
 
-Note that API Gateway doesn't set the `X-Forwarded-Host` header, so we don't trust it by default. 
-You should only whitelist this header if you set it manually, for example in your CloudFront configuration (see how to 
-do it in the [example Cloudformation template](../websites.md#serving-php-and-static-files-via-cloudfront)).
+Note that API Gateway doesn't set the `X-Forwarded-Host` header, so we don't trust it by default. You should only whitelist this header if you set it manually,
+for example in your CloudFront configuration (see how to do it in
+the [example Cloudformation template](../websites.md#serving-php-and-static-files-via-cloudfront)).
 
 > Be careful with these settings if your app will not be executed only in a Lambda environment.
- 
+
 You can get more details in the [Symfony documentation](https://symfony.com/doc/current/deployment/proxies.html).
 
 ### Getting the user IP
 
-**When using CloudFront** on top of API Gateway, you will not be able to retrieve the client IP address and you will
-instead get one of Cloudfront's IP when calling `Request::getClientIp()`. If you really need this, you will need
-to whitelist [every CloudFront IP](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/LocationsOfEdgeServers.html)
+**When using CloudFront** on top of API Gateway, you will not be able to retrieve the client IP address, and you will instead get one of Cloudfront's IP when
+calling `Request::getClientIp()`. If you really need this, you will need to
+whitelist [every CloudFront IP](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/LocationsOfEdgeServers.html)
 in `trusted_proxies`.
 
 ## Assets
 
-To deploy Symfony websites, we need assets to be served by AWS S3. Setting up the S3 bucket is already explained in the [websites documentation](../websites.md#hosting-static-files-with-s3). This section provides additional instructions specific to Symfony assets and Webpack Encore.
+To deploy Symfony websites, assets needs to be served by AWS S3. Setting up the S3 bucket is already explained in
+the [websites documentation](../websites.md#hosting-static-files-with-s3). This section provides additional instructions specific to Symfony assets and Webpack
+Encore.
 
-First, you need to [tell Symfony](https://symfony.com/doc/current/reference/configuration/framework.html#base-urls) to use the S3 URL as the assets base URL, instead of your app domain in production.
+First, you need to [tell Symfony](https://symfony.com/doc/current/reference/configuration/framework.html#base-urls) to use the S3 URL as the assets base URL,
+instead of your app domain in production.
 
 ```yaml
-# config/packages/prod/framework.yaml
+# config/packages/prod/assets.yaml
+
 framework:
   assets:
     base_urls: 'https://<bucket-name>.s3.amazonaws.com'
```

Comment:
Should we really care about S3? Why not do cloudfront directly?

Reply:
This is simpler to setup for beginners and if you put Cloudfront in front of your app, then this is not needed I think.